src/pages/overview/pdf-services-api/quickstarts/java/index.md

```diff
@@ -212,40 +173,55 @@ System.out.println("Exporting " + input_file + " to " + output_file + "\n");
   - `export PDF_SERVICES_CLIENT_ID=<YOUR CLIENT ID>`
   - `export PDF_SERVICES_CLIENT_SECRET=<YOUR CLIENT SECRET>`
 
-6) Next, we can create our credentials and use them:
+5) Next, we can create our credentials and use them:
 
 ```javascript
-// Initial setup, create credentials instance.
-Credentials credentials = Credentials.servicePrincipalCredentialsBuilder()
-        .withClientId(System.getenv("PDF_SERVICES_CLIENT_ID"))
-        .withClientSecret(System.getenv("PDF_SERVICES_CLIENT_SECRET"))
-        .build();
-
-// Create an ExecutionContext using credentials.
-ExecutionContext executionContext = ExecutionContext.create(credentials);
+// Initial setup, create credentials instance
+Credentials credentials = new ServicePrincipalCredentials(
+        System.getenv("PDF_SERVICES_CLIENT_ID"),
+        System.getenv("PDF_SERVICES_CLIENT_SECRET"));
+
+// Creates a PDF Services instance
+PDFServices pdfServices = new PDFServices(credentials);

```

Comment:
The reason, I wanted credentials and PDF services to be together was because Credentials will only be consumed by PDF services therefore it will not make sense to make a separate step

Reply:
As discussed, this seems fine.